src/activeList.js

```diff
@@ -0,0 +1,153 @@
+  const items = document.querySelectorAll('.show-list');
+  items.forEach((item) => {
+    // eslint-disable-next-line eqeqeq
+    if (item.querySelector('.tast').getAttribute('data-time') == itemData.index) {
+      item.querySelector('[data-done]').addEventListener('change', (e) => {
+        e.preventDefault();
+        const itemIndex = itemsArr.indexOf(itemData);
+        const currentItem = itemsArr[itemIndex];
+        // eslint-disable-next-line no-unneeded-ternary
+        currentItem.completed = currentItem.completed ? false : true;
+        itemsArr.splice(itemIndex, 1, currentItem);
+        // eslint-disable-next-line eqeqeq
+        if (currentItem.completed == false) {
+          item.querySelector('.tast').style.textDecoration = '';
+          item.querySelector('.drag-drop').style.display = 'flex';
+          item.querySelector('.trash').style.display = 'none';
+        } else {
+          item.querySelector('.tast').style.textDecoration = 'line-through';
+          item.querySelector('.drag-drop').style.display = 'none';
+          item.querySelector('.trash').style.display = 'flex';
+        }
+        // eslint-disable-next-line consistent-return
+        removeAll.addEventListener('click', (e) => {
+          e.preventDefault();
+          // eslint-disable-next-line eqeqeq
+          if (currentItem.completed == true) {
+            const removeIndex = itemsArr.indexOf(currentItem);
+            itemsArr.splice(removeIndex, 1);
+          }
+          saveLocalStorage(itemsArr);
+        });
+        saveLocalStorage(itemsArr);
+      });
+      item.querySelector('[data-edit]').addEventListener('click', (e) => {
+        e.preventDefault();
+        inputF.value = itemData.description;
+        edit.innerHTML = 'Please Edit Your today\'s To-Do Task here';
+        document.querySelector('#objIndex').value = itemsArr.indexOf(itemData);
+      });
+      item.querySelector('[data-delete]').addEventListener('click', (e) => {
+        e.preventDefault();
+        parentNode.removeChild(item);
+        removeItem(item);
+        saveLocalStorage(itemsArr);
+        // eslint-disable-next-line eqeqeq
+        return itemsArr.filter((item) => item != itemData.description);
+      });
+    }
+  });
+};
+
+const tasksListShow = (itemsArr) => {
+  parentNode.innerHTML = '';
+  if (itemsArr.length > 0) {
+    itemsArr.forEach((item) => {
+      parentNode.insertAdjacentHTML('beforeend',
+        `<div class="show-list">
+        <input type="checkbox" data-done class="check-box" autocomplete="off">
+      <p class="tast" data-edit data-time="${item.index}">${item.description}</p>
+      <img src="images/drag-drop.png" alt="" class="drag-drop">
+      <img src="images/trash.png" alt="" data-delete class="trash">
+      </div>`);
+      handleItem(item);
+    });
+  }
+};
+
+const getItems = () => {
+  const storeItems = localStorage.getItem('itemsArr');
+  if (storeItems === 'undefined' || storeItems == null) {
```

Comment:
I think it's better to use `=== (Triple equals operator)` here

```suggestion
  if (storeItems === 'undefined' || storeItems === null) {
```